docs/src/bot.md

```diff
@@ -42,95 +48,49 @@ snoop_bot(
 )
 ```
 
-[Zygote example](https://github.com/aminya/Zygote.jl/blob/SnoopCompile/deps/SnoopCompile/snoop_bot.jl)
-
-If you do not have additional examples, you can use your runtests.jl file using this syntax:
-
-```julia
-using SnoopCompile
-
-# using runtests:
-snoop_bot( BotConfig("MatLang") )
-```
-
-`snoop_bot` has two variants:
-- `snoopi_bot`: tracks the inference time.
-  This is the default behavior of `snoop_bot` for Julia >= 1.2
-- `snoopc_bot`: tracks the compiling time.
-  This is the default behavior of `snoop_bot` for Julia < 1.2
-
-[Also look at this](https://timholy.github.io/SnoopCompile.jl/stable/snoop/#Precompile-scripts-1)
-----------------------------------
-
-## 3 - Call `snoop_bench` function
+If you choose to use your "runtests.jl" file as your precompile script, configuration can be as simple as specifying just the name of the package:
 
-Call the `snoop_bench` function to measure the effect of adding precompile files. You can also use `snoopv_bench` variant that you uses `timev` for benchmarking.
-
-**Recommendation**:
- - add a `snoop_bench.jl` under `deps/SnoopCompile` (or at the same path that is referenced in yaml file).
- - or call the function directly from the yaml file.
-
-Benchmarking the inference time of example_script
 ```julia
-
 using SnoopCompile
 
-botconfig = BotConfig(
-  "Zygote";
-  os = ["linux", "windows", "macos"],
-  version = [v"1.4.1", v"1.3.1"],
-  blacklist = ["SqEuclidean"],
-  exhaustive = false,
-)
-
-println("Benchmarking the inference time of example_script")
-snoop_bench(
-  botconfig,
-  "$(@__DIR__)/example_script.jl",
-)
+snoop_bot(BotConfig("MyPackage"))
 ```
 
-Benchmarking the inference time of the tests
-```julia
-println("Benchmarking the inference time of the tests")
-snoop_bench(BotConfig("MatLang"))
-```
+Finally, you could use package loading as the only operation,
+with `snoop_bot(config, :(using MyPackage))`.
 
-To selectively exclude some of your tests from running by SnoopCompile bot, use the global SnoopCompile_ENV::Bool variable.
-```julia
-if !isdefined(Main, :SnoopCompile_ENV) || SnoopCompile_ENV == false
-  # the tests you want to skip in SnoopCompile environment
-end
-```
```

Comment:
We can probably edit this for SnoopCompile_ENV documentation. I will take a look when I find some time.

```
To selectively exclude some of your tests from running by SnoopCompile bot, use the global SnoopCompile_ENV::Bool variable.	`snoop_bot` uses different strategies depending on the Julia version:
```julia	
if !isdefined(Main, :SnoopCompile_ENV) || SnoopCompile_ENV == false	
  # the tests you want to skip in SnoopCompile environment	
end	
```
```